lsr_role2collection/collection_readme.md

```diff
@@ -0,0 +1,50 @@
+﻿# Linux System Roles Ansible Collection
+Linux System Roles is a collection of roles for managing Linux system components.
+
+## Currently supported distributions
+* Fedora
+* Red Hat Enterprise Linux (RHEL 6+)
+* RHEL 6+ derivatives such as CentOS 6+
+
+NOTE: Some roles are not supported in RHEL6 and RHEL7. For more details about the individual roles you are interested in, see the documentation.
+
+## Dependencies
+
+The following dependency is required for the Ansible Controller:
+* python3-jmespath

```

Comment:
I think we should just say `jmespath`
Otherwise, we'll need to say all of the different names for the package e.g. `python2-jmespath` on EL7, `python3-jmespath` on EL8+/Fedora, `jmespath` if doing a `pip` install, etc.